notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<p>Prefer a path relative to the position of the notebook in the repo rather than <code>~/asl-ml-immersion</code> so that the command does not require the repo to be cloned into a user home directory. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I agree. Modified.</p>